README.md

```diff
@@ -61,40 +61,44 @@ There are totally 27 workloads in HiBench. The workloads are divided into 6 cate
 2. K-means clustering (Kmeans)
 
     This workload tests the K-means (a well-known clustering algorithm for knowledge discovery and data mining) clustering in spark.mllib. The input data set is generated by GenKMeansDataset based on Uniform Distribution and Guassian Distribution. There is also an optimized K-means implementation based on DAL (Intel Data Analytics Library), which is available in the dal module of sparkbench.
+    
+3. Gaussian Mixture Model (GMM) 
```

Comment:
Could you also update the root README `There are totally {XX} workloads in HiBench`?